src/mui/form/FormInput.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import FormField from './FormField';
 
+const defaultStyle = {
+    minWidth: 'calc(50% - 8px)',
+    maxWidth: 'calc(50% - 8px)',
+};
+
 const FormInput = ({ input, ...rest }) =>
     input ? (
         <div
             className={`aor-input aor-input-${input.props.source}`}
-            style={input.props.style}
+            style={input.props.style ? input.props.style : defaultStyle}
```

Comment:
optional: input.props.style || defaultStyle could be lighter (our linter corrects it this way)